shaptools/shapcli.py

```diff
@@ -0,0 +1,353 @@
+"""
+Code to expose some useful methods using the command line
+
+:author: xarbulu
+:organization: SUSE LLC
+:contact: xarbulu@suse.com
+
+:since: 2019-07-11
+"""
+
+import logging
+import argparse
+import json
+
+from shaptools import hana
+
+PROG = 'shapcli'
+LOGGING_FORMAT = '%(message)s'
+
+
+class DecodedFormatter(logging.Formatter):
```

Comment:
I was asking me if we need a Class here, and we could not achieving it in a more functional way without it.

I think that this is perhaps the more pythonic way but I was thinking if we could just use an helper function for setting the  logging format. 

It is more a curiosity from the functional world. I"m also ok to have this with class and super

Reply:
I created a class because the standard formatter itself is an object/class (actually in python everything is an object...). The unique thing I needed was to reformat the captured logs to remote the annoying `b''` part from the logs. As far as I know this is how the python logging works.
I'm afraid that is not easy to apply functional programming techniques here hehe